documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
+
+## Goals
+Goals of conducted tests are as following:
+* Assess system performance,
+* Assess system stability.
+
+### Metrics
+During our tests we focus on following metrics and correlate it to came out with observations:
+* throughput,
+* response time - 90% Line,
+* quantity of Virtual Users,
+* Number of errors (HTTP response codes > 400, Knotx logs),
+* infrastructure resources (cpu, memory, disk, network),
+* application resources (jmx heap size, garbage collections).
+
+### KPIs
+* 1 second for response time (90%Line),
+* error occurrences.
+* stability:
+    * No memory leaks
+    * No errors (logs, responses),
+    * Infrastructure resources OK,
+    * Application resources OK.
+	
+## Test Environment
+Whole Test Environment is set up on AWS Cloud.
+<p align="center">
+  <img align="center"
+    src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-test-environment.png?raw=true"
+    alt="Basic request flow"/>
+</p>
+
+### System under test
+* 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8,
+* 1 x Knotx - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8,
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8,
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
+
+### Test harness
+* 1 x Jmeter Master - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8,
+* 2 x Jmeter Servers - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+
+### Repositories
+* mocks-knotx - https://stash.cognifide.com/projects/OCTPI/repos/mocks-knotx/browse
```

Comment:
We can't link to cognifide stash here :) we'd need to move it to this repo. e.g. /performance-tests/mocks-knotx and /performance-tests/config-knotx